tiledb/test-utils/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 extern crate proptest;
 
+pub mod signal;
```

Comment:
We should probably make this a `#[cfg(!windows)]` or whatever. I know we're not building there yet, but it'll be a requirement soonishly. Or perhaps a feature flag as it seems like the sort of thing we'd only enable as a tool for debugging something that's failing? Not sure on that one. I might be nice to have something that attempts to print a stack on any signal encountered during test runs.

Reply:
Resolution: "signal" feature, and the interior implementation is behind `#[cfg(not(windows))]` so that the API is the same regardless of platform.